Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tabbycat"
-version = "0.1.3"
+version = "0.1.4"
```

Comment:
Since I haven't publish the `0.1.3`, I think this version bump is not needed. We just can put all these changes in one version?

Reply:
👍🏼 78d7ba8